src/components/visualizations/chart/test/chartOptionsBuilder.spec.ts

```diff
@@ -2312,16 +2313,40 @@ describe("chartOptionsBuilder", () => {
             });
         });
 
-        describe("stackMeasuresToPercent", () => {
-            const tooltipFn = buildTooltipFactory(null, "column", { stackMeasuresToPercent: true });
-            it.each([["0%", 0], ["49.01%", 49.0111], ["100%", 100]])(
-                "should render %s when percentageValue is %s",
-                (formattedValue: string, percentageValue: number) => {
-                    const tooltip = tooltipFn(pointData, percentageValue);
-                    expect(getValues(tooltip)).toEqual(["series", formattedValue]);
-                },
-            );
-        });
+        it.each([
+            ["number", false, false, false, " 1"],
+            ["number", false, false, true, " 1"],
+            ["number", false, true, false, " 1"],
+            ["number", false, true, true, " 1"],
+            ["percent", true, false, false, "49.01%"],
+            ["percent", true, false, true, "49.01%"],
+            ["percent", true, true, false, "49.01%"],
+            ["number", true, true, true, " 1"],
+        ])(
+            "should render %s when stackMeasuresToPercent is %s, isDualAxis is %s and isSecondAxis is %s",
+            (
+                _expectededText: string,
```

Comment:
we have typo `_expectededText` here.
I suggest to rename it to `_type`

Reply:
updated